README.md

```diff
@@ -8,23 +8,58 @@ Alpakka [![scaladex-badge][]][scaladex] [![travis-badge][]][travis] [![gitter-ba
 [gitter]:                    https://gitter.im/akka/akka
 [gitter-badge]:       https://badges.gitter.im/akka/akka.svg
 
-This project provides a home to Akka Streams connectors to various technologies, protocols or libraries.
+Systems don't come alone. In the world of microservices, cloud deployment and services, and our history of existing solutions we need to integrate. [Reactive Streams](http://www.reactive-streams.org/) give us a technology-independent tool to let these systems communicate without overwhelming each other.
+Alpakka's connectors let you stream data in and out of other technologies using [Akka's](https://doc.akka.io/docs/akka/current/stream/index.html) implementation of the Reactive Streams specification. 
+
 
 Documentation
 -------------
 
-See our [documentation](http://developer.lightbend.com/docs/alpakka/current/) and [scaladoc](http://developer.lightbend.com/docs/api/alpakka/current/akka/stream/alpakka/index.html) pages.
+The **Alpakka reference documentation** is available at [Lightbend Tech Hub](http://developer.lightbend.com/docs/alpakka/current/).
+
+The **Alpakka Kafka connector documentation** is available at [akka.io](https://doc.akka.io/docs/akka-stream-kafka/current/).
+
+To get a grip of the latest Alpakka releases check out [Alpakka releases](https://github.com/akka/alpakka/releases) and [Alpakka Kafka connector releases](https://github.com/akka/reactive-kafka/releases).
+
+
+Community
+---------
+
+You can join these groups and chats to discuss and ask Akka and Alpakka related questions:
+
+- Forums: [discuss.lightbend.com](https://discuss.lightbend.com/c/akka)
```

Comment:
name: discuss.akka.io

Reply:
Link to alpakka subforum directly, not top level Akka

Reply:
Yup. Linking to https://discuss.lightbend.com/c/akka/streams-and-alpakka would be nice.

Reply:
Why write discuss.akka.io? Its redirected anyway - and akka.io doesn't deep-link.